R/utils.R

```diff
@@ -30,13 +28,6 @@ vcapply <- function(x, FUN, ...) {
   vapply(x, FUN, FUN.VALUE = character(1), ...)
 }
 
-release_bullets <- function() {
-  c(
-    '`usethis::use_latest_dependencies(TRUE, "CRAN")`',
-    NULL
-  )
-}
-
```

Comment:
We do NOT want to remove this.

This function is consulted by `devtools::release()` and `usethis::use_release_issue()`.

Reply:
Ah, sorry I missed that `{usethis}` is accessing these internal objects from `{devtools}`.

Is this where this function is used?

https://github.com/r-lib/usethis/blob/8f045dc0a88eac82a602df56e2dd62df90971a04/R/release.R#L133-L135

At any rate, I have reverted this change.